posthog/src/main/java/com/posthog/internal/PostHogApi.kt

```diff
@@ -11,12 +11,14 @@ import java.io.IOException
 import java.io.OutputStream
 import java.util.Date
 
+/**
+ * The class that calls the PostHog API
+ * @property config the Config
```

Comment:
I wouldn't have placeholder comments like these... 

I think the comments here should be to help the busy person who has just added PostHog and doesn't know what to do

"this is the class PostHog uses to blah, it shouldn't be used directly, if you want to wotsit, you need a clinkexpander instead"

(although _maybe_ not that exact text 🤣)

Reply:
Yeah totally agree with you but those classes are all internal, they are not visible to clients at all, so its mostly for us.

Reply:
> so its mostly for us.
Yep, so I guess the useful comments then are when someone over confident like me thinks "Manoel is on holiday - I'll do a PR for this". What does the internal implementer really need to know here?

There's always the risk of a comment I genuinely saw in a previous job (pseudo code for easiness of typing)

```
// the person's name
// string
public getAge int
```